src/utils/Validations.tsx

```diff
@@ -21,19 +23,15 @@ export const validateRegisterData = (registerData: any) => {
 	if (!registerData.email || registerData.email.length === 0) {
 		validateErrors.email = EMAIL_EMPTY
 		validateErrors.isError = true
-	}
-
-	if (!validateEmail(registerData.email)) {
+	} else if (!validateEmail(registerData.email)) {
 		validateErrors.email = EMAIL_INVALID
 		validateErrors.isError = true
 	}
 
 	if (!registerData.password || registerData.password.length === 0) {
 		validateErrors.password = PASSWORD_EMPTY
 		validateErrors.isError = true
-	}
-
-	if (registerData.password && registerData.password.length < 6) {
+	} else if (registerData.password && registerData.password.length < 6) {
```

Comment:
~~ the check for `registerData.password` existing is redundant - if it didn't, you would have landed in the first condition